eslint error stops the build: `Parsing error: Identifier 'list' has already been declared`

sorry about the critical status :sweat: thanks in advance for looking into it!

full error:
```
ERROR in ./jsapp/js/components/modalForms/submission.es6
Module Error (from /srv/node_modules/eslint-loader/index.js):

/srv/src/kpi/jsapp/js/components/modalForms/submission.es6
  273:15  error  Parsing error: Identifier 'list' has already been declared

  271 |         break;
  272 |       case 'begin_repeat':
> 273 |         const list = submissionValue.map((r) => {
      |               ^
  274 |           const stringified = JSON.stringify(r);
  275 |           return <li key={stringified}>{stringified}</li>
  276 |         });

 1 problem (1 error, 0 warnings)
```
(there's [nothing new at all](https://github.com/kobotoolbox/kpi/blame/d29f374c238dcf93a2b35bdbab000863f2e41dac/jsapp/js/components/modalForms/submission.es6#L272) about this `const list`)

crappy workaround:
```diff
kpi$ git diff
diff --git a/webpack/webpack.common.js b/webpack/webpack.common.js
index 51d99b00..8a4171e4 100644
--- a/webpack/webpack.common.js
+++ b/webpack/webpack.common.js
@@ -23,12 +23,12 @@ const postCssLoader = {
 var defaultOptions = {
   module: {
     rules: [
-      {
+      /*{
         enforce: "pre",
         test: /\.(js|jsx|es6)$/,
         exclude: /node_modules/,
         loader: 'eslint-loader',
-      },
+      },*/
       {
         test: /\.(js|jsx|es6)$/,
         exclude: /node_modules/,
```

https://www.flowdock.com/app/kobotoolbox/kobo/threads/6wnm_1A0IG1KkwTkF8kDCdpPFLf

--- a/.eslintrc
+++ b/.eslintrc
@@ -34,7 +34,7 @@
     "react/react-in-jsx-scope": 2,
     "react/self-closing-comp": 2,
     "react/wrap-multilines": 0,
-    "no-trailing-spaces":0,
+    "no-trailing-spaces": 0,
     "no-irregular-whitespace": 0,
     "no-mixed-spaces-and-tabs": 0,
     "no-unused-expressions": 0,
@@ -47,15 +47,16 @@
     "no-undef": 0,
     "eqeqeq": 0,
     "no-multi-spaces": 0,
-    "no-extend-native":0,
-    "no-script-url":0,
-    "no-extra-bind":0,
-    "semi-spacing":0,
-    "no-extra-semi":0,
-    "no-extra-strict":0,
-    "no-new-object":0,
-    "no-debugger":0
-
+    "no-extend-native": 0,
+    "no-script-url": 0,
+    "no-extra-bind": 0,
+    "semi-spacing": 0,
+    "no-extra-semi": 0,
+    "no-extra-strict": 0,
+    "no-new-object": 0,
+    "no-debugger": 0,
+    "no-case-declarations": 1,
+    "no-fallthrough": 1
   },
   "globals": {
     "inject": false,
@@ -69,7 +70,7 @@
     "expect": false,
     "window": false,
     "document": false,
-    "Parse":false
+    "Parse": false
   },
   "plugins": [
     "react"

--- a/jsapp/js/components/modalForms/submission.es6
+++ b/jsapp/js/components/modalForms/submission.es6
@@ -238,7 +238,6 @@ class Submission extends React.Component {
   responseDisplayHelper(q, s, overrideValue = false, name) {
     if (!q) return false;
     const choices = this.props.asset.content.choices;
-    let list;
     let translationIndex = this.state.translationIndex;
 
     var submissionValue = s[name];
@@ -247,16 +246,17 @@ class Submission extends React.Component {
       submissionValue = overrideValue;
 
     switch(q.type) {
-      case 'select_one':
+      case 'select_one': {
         const choice = choices.find(x => x.list_name == q.select_from_list_name && x.name === submissionValue);
         if (choice && choice.label && choice.label[translationIndex])
           return choice.label[translationIndex];
         else
           return submissionValue;
         break;
-      case 'select_multiple':
+      }
+      case 'select_multiple': {
         var responses = submissionValue.split(' ');
-        list = responses.map((r)=> {
+        var list = responses.map((r)=> {
           const choice = choices.find(x => x.list_name == q.select_from_list_name && x.name === r);
           if (choice && choice.label && choice.label[translationIndex])
             return <li key={r}>{choice.label[translationIndex]}</li>;
@@ -265,21 +265,25 @@ class Submission extends React.Component {
         })
         return <ul>{list}</ul>;
         break;
+      }
       case 'image':
       case 'audio':
-      case 'video':
+      case 'video': {
         return this.renderAttachment(submissionValue, q.type);
         break;
-      case 'begin_repeat':
-        list = submissionValue.map((r) => {
+      }
+      case 'begin_repeat': {
+        const list = submissionValue.map((r) => {
           const stringified = JSON.stringify(r);
           return <li key={stringified}>{stringified}</li>
         });
         return <ul>{list}</ul>
         break;
-      default:
+      }
+      default: {
         return submissionValue;
         break;
+      }
     }
   }
   renderRows() {

--- a/jsapp/js/components/modalForms/submission.es6
+++ b/jsapp/js/components/modalForms/submission.es6
@@ -238,6 +238,7 @@ class Submission extends React.Component {
   responseDisplayHelper(q, s, overrideValue = false, name) {
     if (!q) return false;
     const choices = this.props.asset.content.choices;
+    let list;
     let translationIndex = this.state.translationIndex;
 
     var submissionValue = s[name];
@@ -255,7 +256,7 @@ class Submission extends React.Component {
         break;
       case 'select_multiple':
         var responses = submissionValue.split(' ');
-        var list = responses.map((r)=> {
+        list = responses.map((r)=> {
           const choice = choices.find(x => x.list_name == q.select_from_list_name && x.name === r);
           if (choice && choice.label && choice.label[translationIndex])
             return <li key={r}>{choice.label[translationIndex]}</li>;
@@ -270,7 +271,7 @@ class Submission extends React.Component {
         return this.renderAttachment(submissionValue, q.type);
         break;
       case 'begin_repeat':
-        const list = submissionValue.map((r) => {
+        list = submissionValue.map((r) => {
           const stringified = JSON.stringify(r);
           return <li key={stringified}>{stringified}</li>
         });

--- a/jsapp/js/dataInterface.es6
+++ b/jsapp/js/dataInterface.es6
@@ -32,10 +32,10 @@ var dataInterface;
     if (request.status === 403 || request.status === 401 || request.status === 404) {
       dataInterface.selfProfile().done((data) => {
         if (data.message === 'user is not logged in') {
-          let errorMessage = t("Please try reloading the page. If you need to contact support, note the following message: <pre>##server_message##</pre>")
+          let errorMessage = t('Please try reloading the page. If you need to contact support, note the following message: <pre>##server_message##</pre>')
           let serverMessage = request.status.toString();
           if (request.responseJSON && request.responseJSON.detail) {
-            serverMessage += ": " + request.responseJSON.detail;
+            serverMessage += ': ' + request.responseJSON.detail;
           }
           errorMessage = errorMessage.replace('##server_message##', serverMessage);
           alertify.alert(t('You are not logged in'), errorMessage);

--- a/jsapp/js/mixins.es6
+++ b/jsapp/js/mixins.es6
@@ -236,7 +236,7 @@ const applyImport = (params) => {
           uid: data.uid,
         }).done((importData) => {
           switch (importData.status) {
-            case 'complete':
+            case 'complete': {
               const finalData = importData.messages.updated || importData.messages.created;
               if (finalData && finalData.length > 0 && finalData[0].uid) {
                 clearInterval(doneCheckInterval);
@@ -246,14 +246,17 @@ const applyImport = (params) => {
                 reject(importData);
               }
               break;
+            }
             case 'processing':
-            case 'created':
+            case 'created': {
               // TODO: notify promise awaiter about delay (after multiple interval rounds)
               break;
+            }
             case 'error':
-            default:
+            default: {
               clearInterval(doneCheckInterval);
               reject(importData);
+            }
           }
         }).fail((failData)=>{
           clearInterval(doneCheckInterval);

--- a/package.json
+++ b/package.json
@@ -82,6 +82,7 @@
     "build-production": "npm run build && echo 'WARNING: npm run build-production DEPRECATED! PLEASE USE npm run build INSTEAD'",
     "watch": "webpack-dev-server --config webpack/dev.server.js",
     "test": "webpack --config webpack/test.config.js --progress --colors && mocha-chrome test/tests.html",
+    "lint": "eslint --ext .es6 \"jsapp/js/**/*\"",
     "show-icons": "opn jsapp/fonts/k-icons.html",
     "generate-icons": "node ./scripts/generate_icons.js",
     "copy-fonts": "python ./scripts/copy_fonts.py && npm run generate-icons"

